10:38:20 - ERR - getRecords - (IndexError('tuple index out of range'),)
10:39:52 - ERR - getRecords - (IndexError('tuple index out of range'),)
10:51:39 - ERR - get_users - (AttributeError("'NoneType' object has no attribute 'append'"),)
